src/webgpu/api/validation/image_copy/README.txt

```diff
@@ -9,6 +9,7 @@ Test coverage:
 	- texture_must_be_valid: for valid, destroyed, error textures.
 	- sample_count_must_be_1: for sample count 1 and 4.
 	- mip_level_must_be_in_range: for various combinations of mipLevel and mipLevelCount.
+	- full_copy_for_depth_stencil_format: for full and non-full copies on width, height, and depth.
```

Comment:
Test name doesn't match. I think the others might not match anymore either, whoops.

FWIW it's not super necessary that we keep this README up to date. I'm supposed to dissolve this README out into proper per-test descriptions, but I haven't done it yet. But better to make this one thing match now so it'll be easier to dissolve later.

Reply:
Done. Now that the document is still there, I'd like to make it consistent with tests. We can remove the file (and move description to each test if we haven't) after all tests related to copy validation are done.